### ISSUE TITLE
Not to check credentials when dry run only

### ISSUE THREAD

**Issue by AlexV525** [sakebook/actions-flutter-pub-publisher#28]
When we run dry run on forks, it always checks credentials on the forked repo, which causes checks to fail since the forked repo typically does not have a secret of the pub.

It's better we bypass verify steps when users run dry run only.

**Comment by alexcmgit** [comment]
Not really, what if someone want to run `dry-run: true` to check if the credentials are valid?

Better we create a new option like `dry-run-bypass-credentials: true` rather than modifying this current behavior.

**Comment by AlexV525** [comment]
As I concern, credentials doesn't do anything during the dry run. So it should not be checked.

**Comment by alexcmgit** [comment]
Oh I see now. In this case why not just create an empty repo var?

**Comment by AlexV525** [comment]
Not sure if you're indicating a solution.

When forks are submitting PRs, we might use dry-run for them. Since their repo is missing the credential, checks will failed immediately because the credential check is placed at the first place.
